app/controllers/search_controller.rb

```diff
@@ -46,8 +46,6 @@ def quota_search
 
     form = QuotaSearchForm.new(params)
     @result = QuotaSearchPresenter.new(form)
-    @enable_service_switch_banner_in_action = false
```

Comment:
Shouldn't you replace this with a call to `disable_switch_service_banner`

Reply:
Good catch! 
Actually here the banner was enabled here, so I removed that line.

(it has been enabled in various places too)